app/src/main/java/com/pavesid/androidacademy/services/MoviesWorkerRepository.kt

```diff
@@ -0,0 +1,14 @@
+package com.pavesid.androidacademy.services
+
+import androidx.work.Constraints
+import androidx.work.NetworkType
+import androidx.work.PeriodicWorkRequestBuilder
+import java.util.concurrent.TimeUnit
+
+object MoviesWorkerRepository {
+    private val constraints = Constraints.Builder().setRequiresCharging(true).setRequiredNetworkType(NetworkType.CONNECTED).build()
```

Comment:
The assignment says - "when he connected to unmetered network"

Reply:
Done.